vendor/bat-native-ads/src/bat/ads/internal/ads_impl.h

```diff
@@ -253,6 +253,11 @@ class AdsImpl : public Ads {
   bool IsCatalogOlderThanOneDay();
   void NotificationAllowedCheck(
       const bool serve);
+	  
+  #if defined(OS_ANDROID)
+  void RemoveAllNotificationsAfterReboot();
+  void RemoveAllNotificationsAfterUpdate();
```

Comment:
nit: change we change `Update` to `Upgrade`

Reply:
If we are going that way, I would like to see a document that describes all that requirements for the code. Put it in the repo, put it in the wiki.

Reply:
NONCRITICAL
Covered by: brave/brave-browser#6235 (9)